wrapanapi/containers/providers/rhopenshift.py

```diff
@@ -217,14 +246,22 @@ def deploy_template(self, template, tags=None, password='smartvm', **kwargs):
         self.create_project(name=proj_name)
         progress_callback("Created Project `{}`".format(proj_name))
 
+        version = retrieve_cfme_appliance_version(template)
+
         # grant rights according to scc
         self.logger.info("granting rights to project %s sa", proj_name)
-        scc_user_mapping = (
-            {'scc': 'anyuid', 'user': 'cfme-anyuid'},
-            {'scc': 'anyuid', 'user': 'cfme-orchestrator'},
-            {'scc': 'anyuid', 'user': 'cfme-httpd'},
-            {'scc': 'privileged', 'user': 'cfme-privileged'},
-        )
+        if version >= '5.9':
```

Comment:
**QUESTION**are we ok with a string compare here, (i suspect yes, but its irky without a comment/version class instance)

Reply:
I believe it is ok for now. I'll delete all these if statements soon when we stop testing 5.8.